alonzo/test/test/Test/Cardano/Ledger/Alonzo/Trials.hs

```diff
@@ -294,19 +294,19 @@ searchForQuickFailure seconds seed = do
       ( localOption
           (QuickCheckReplay (Just seed))
           -- some properties we might use
-          -- (testProperty "preserves ADA" $ adaPreservationChain @(AlonzoEra TestCrypto))
+          (testProperty "preserves ADA" $ adaPreservationChain @(AlonzoEra TestCrypto))
           -- (testProperty "Delegation Properties" (delegProperties @(AlonzoEra TestCrypto)))
           -- fastPropertyTests
           -- (propertyTests @(AlonzoEra TestCrypto))
-          (testProperty "Only valid CHAIN STS signals are generated" (onlyValidLedgerSignalsAreGenerated @(AlonzoEra TestCrypto)))
+          -- (testProperty "Only valid CHAIN STS signals are generated" (onlyValidLedgerSignalsAreGenerated @(AlonzoEra TestCrypto)))
```

Comment:
Did you mean to comment this one out? Perhaps `preserves ADA` is good enough for the fast tests?

Reply:
ah, nm, this is only for debugging. looks helpful!